README.md

```diff
@@ -83,3 +83,6 @@ If you utilize bind mounts instead of Docker named volumes (e.g. - `-v /path/to/
 chown -R 508:508 /data/omada/data /data/omada/work /data/omada/logs
 ```
 
+## Custom Certificates
+
+By default, Omada software uses self-signed certificates. If however you want to use custom certificates you can mount them in /cert/tls.key and /cert/tls.crt. This is compatible with kubernetes TLS secrets. Entrypoint script will convert them into Java Keystore used by jetty inside the Omada SW.
```

Comment:
One thing that might be useful would be describing what the expected format and stacking order of the certs would be in the `tls.crt` file.  Is it the server cert, intermediate cert(s), and then the CA cert?

Reply:
Good ide. Update.